test/broadcast.jl

```diff
@@ -1056,3 +1056,8 @@ end
     @test Broadcast.BroadcastFunction(+)(2:3, 2:3) == 4:2:6
     @test Broadcast.BroadcastFunction(+)(2:3, 2:3) isa AbstractRange
 end
+
+@testset "#42063" begin
+    A = println.(stdout, [1,2,3,4])
```

Comment:
Printing to `stdout` will just add noise to the output when testing. You could print to an `IOBuffer` instead and check that the resulting content is correct.

Reply:
I guess IOBuffer works even without this pull request.

Reply:
Does `IOBuffer` work for you without this PR? For me it doesn’t:
```
julia> println.(IOBuffer(), [1,2,3])
ERROR: MethodError: no method matching length(::IOBuffer)
```

Reply:
Ah, I see. You tried to print to the *type* `IOBuffer`, not to an instance of an `IOBuffer`.

Reply:
my bad, I forgot to add those parentheses. You are right, it does not.

Reply:
I meant something like this:
```julia
io = IOBuffer()
println.(io, [1,2,3])
@test String(take!(io)) == "1\n2\n3\n"
```